api_server/src/request/actions.rs

```diff
@@ -18,6 +18,7 @@ enum ActionType {
     BlockDeviceRescan,
     FlushMetrics,
     InstanceStart,
+    SendCtrlAltDel,
```

Comment:
This needs to be documented in docs/api_requests/actions.md

Reply:
Done.